src/http/api/views/account.posts.view.ts

```diff
@@ -90,16 +92,14 @@ export class AccountPostsView {
         currentContextAccount: Account,
         limit: number,
         cursor: string | null,
-    ): Promise<Result<AccountPosts, GetPostsError>> {
+    ): Promise<Result<AccountPosts, GetPostsError | GetPostsFromOutboxError>> {
```

Comment:
This should never return a `GetPostsFromOutboxError` I think - because this method should support both internal and external accounts. 

I don't think the caller of this method is in a position to be able to handle a `GetPostsFromOutboxError` - it means there's a bug in our code if they get one.

I think we need to check the return of the call to `getPostsFromOutbox` on line 98 - and if it's an error - we can `throw` an error, because at that point we're in an undefined state?